### PR TITLE
Add padding CSS properties compat data

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds compat data for the following CSS properties:

* [`padding`](https://developer.mozilla.org/docs/Web/CSS/padding)
* [`padding-left`](https://developer.mozilla.org/docs/Web/CSS/padding-left)
* [`padding-right`](https://developer.mozilla.org/docs/Web/CSS/padding-right)
* [`padding-bottom`](https://developer.mozilla.org/docs/Web/CSS/padding-bottom)
* [`padding-top`](https://developer.mozilla.org/docs/Web/CSS/padding-top)
* [`padding-block-end`](https://developer.mozilla.org/docs/Web/CSS/padding-block-end)
* [`padding-block-start`](https://developer.mozilla.org/docs/Web/CSS/padding-block-start)
* [`padding-inline-end`](https://developer.mozilla.org/docs/Web/CSS/padding-inline-end)
* [`padding-inline-start`](https://developer.mozilla.org/docs/Web/CSS/padding-inline-start)

### CODE REVIEW

File: css/properties/padding-bottom.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-bottom.json]: "4"

File: css/properties/padding-bottom.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "padding-bottom": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding-bottom",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-bottom.json]: true 
padding-* are such basic properties, that all supports it (but eventually Opera Mini but we don't track it)

File: css/properties/padding-bottom.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-bottom.json]: true

File: css/properties/padding-bottom.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": "4"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-bottom.json]: true

File: css/properties/padding-bottom.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "opera": {
+              "version_added": "3.5"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-bottom.json]: true

File: css/properties/padding-bottom.json [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "safari": {
+              "version_added": "1"
+            },
+            "safari_ios": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-bottom.json]: true

File: css/properties/padding-block-end.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": true
```
Comment (teoli2003) [inline css/properties/padding-block-end.json]: Where does this come from?

false

File: css/properties/padding-block-end.json [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-block-end.json]: false

File: css/properties/padding-block-end.json [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding-block-end",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-block-end.json]: false

File: css/properties/padding-inline-end.json [hunk 10]
```diff
@@ -0,0 +1,89 @@
+{
+  "css": {
+    "properties": {
+      "padding-inline-end": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding-inline-end",
+          "support": {
+            "webview_android": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/padding-inline-end.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-inline-end.json [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "chrome": {
+              "alternative_name": "-webkit-padding-end",
+              "version_added": "2"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-inline-end.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-inline-end.json [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/padding-inline-end.json]: Supports the alternative_name w/ version_added: 15

File: css/properties/padding-inline-end.json [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/padding-inline-end.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-inline-end.json [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "safari": {
+              "alternative_name": "-webkit-padding-end",
+              "version_added": "3"
+            },
+            "safari_ios": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/padding-inline-end.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-inline-start.json [hunk 15]
```diff
@@ -0,0 +1,89 @@
+{
+  "css": {
+    "properties": {
+      "padding-inline-start": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding-inline-start",
+          "support": {
+            "webview_android": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/padding-inline-start.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-inline-start.json [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "chrome": {
+              "alternative_name": "-webkit-padding-start",
+              "version_added": "2"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-inline-start.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-inline-start.json [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "edge": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-inline-start.json]: false

File: css/properties/padding-inline-start.json [hunk 18]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "edge_mobile": {
+              "version_added": true
```
Comment (teoli2003) [inline css/properties/padding-inline-start.json]: false

File: css/properties/padding-inline-start.json [hunk 19]
```diff
@@ -0,0 +1,89 @@
...
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/padding-inline-start.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-inline-start.json [hunk 20]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/padding-inline-start.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-inline-start.json [hunk 21]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "safari": {
+              "alternative_name": "-webkit-padding-start",
+              "version_added": "3"
+            },
+            "safari_ios": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/padding-inline-start.json]: Supports the alternative_name w/ version_added: true

File: css/properties/padding-left.json [hunk 22]
```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "padding-left": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding-left",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-left.json]: true

File: css/properties/padding-left.json [hunk 23]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-left.json]: true

File: css/properties/padding-left.json [hunk 24]
```diff
@@ -0,0 +1,57 @@
...
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-left.json]: "4"

File: css/properties/padding-left.json [hunk 25]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "ie": {
+              "version_added": "4"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-left.json]: true

File: css/properties/padding-left.json [hunk 26]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "opera": {
+              "version_added": "3.5"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-left.json]: true

File: css/properties/padding-left.json [hunk 27]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "safari": {
+              "version_added": "1"
+            },
+            "safari_ios": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-left.json]: true

File: css/properties/padding-right.json [hunk 28]
```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "padding-right": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding-right",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-right.json]: true

File: css/properties/padding-right.json [hunk 29]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-right.json]: true

File: css/properties/padding-right.json [hunk 30]
```diff
@@ -0,0 +1,57 @@
...
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-right.json]: "4"

File: css/properties/padding-right.json [hunk 31]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "ie": {
+              "version_added": "4"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-right.json]: true

File: css/properties/padding-right.json [hunk 32]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "opera": {
+              "version_added": "3.5"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-right.json]: true

File: css/properties/padding-right.json [hunk 33]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "safari": {
+              "version_added": "1"
+            },
+            "safari_ios": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-right.json]: true

File: css/properties/padding-top.json [hunk 34]
```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "padding-top": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding-top",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-top.json]: true

File: css/properties/padding-top.json [hunk 35]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-top.json]: true

File: css/properties/padding-top.json [hunk 36]
```diff
@@ -0,0 +1,57 @@
...
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-top.json]: "4"

File: css/properties/padding-top.json [hunk 37]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "ie": {
+              "version_added": "4"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-top.json]: true

File: css/properties/padding-top.json [hunk 38]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "opera": {
+              "version_added": "3.5"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-top.json]: true

File: css/properties/padding-top.json [hunk 39]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "safari": {
+              "version_added": "1"
+            },
+            "safari_ios": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding-top.json]: true

File: css/properties/padding.json [hunk 40]
```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "padding": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding.json]: true

File: css/properties/padding.json [hunk 41]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding.json]: true

File: css/properties/padding.json [hunk 42]
```diff
@@ -0,0 +1,57 @@
...
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding.json]: "4"

File: css/properties/padding.json [hunk 43]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "ie": {
+              "version_added": "4"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding.json]: true

File: css/properties/padding.json [hunk 44]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "opera": {
+              "version_added": "3.5"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding.json]: true

File: css/properties/padding.json [hunk 45]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "safari": {
+              "version_added": "1"
+            },
+            "safari_ios": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/padding.json]: true

**Comment by ddbeck** [comment]
OK, that was pretty easy. Let me know if you want to see any more changes. 😄

**Pull request merged**